### PR TITLE
add unit test for helper.go

### PR DESCRIPTION
**Pull request by nayihz**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
add unit test for helper.go

**Which issue(s) this PR fixes**:
add unit test

### CODE REVIEW

**Comment by latteczy** [comment]
hi @smarterclayton @caesarxuchao @yujuhong , can anybody add ok-to-test label for this pr? Thanks.

File: pkg/apis/core/v1/helper/helpers_test.go [hunk 1]
```diff
@@ -1509,3 +1509,93 @@ func TestHugePageUnitSizeFromByteSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLoadBalancerStatusEqual(t *testing.T) {
+
+	testCases := []struct {
```
Comment (oomichi) [inline pkg/apis/core/v1/helper/helpers_test.go:1515]: It is better to put `name` for each test pattern and output the name at line 1597 when the pattern is failed.

**Review by oomichi — COMMENTED** [review]
LoadBalancerStatusEqual() is not covered on unit tests yet, thanks for doing this.
Just one comment about error message.

/cc @oomichi

File: pkg/apis/core/v1/helper/helpers_test.go (hunk 1 above)
Comment (latteczy) [reply pkg/apis/core/v1/helper/helpers_test.go]: https://github.com/kubernetes/kubernetes/pull/96353

**Pull request closed without merging**